docroot/sites/default/settings/default.local.settings.php

```diff
@@ -7,6 +7,16 @@
 
 use Drupal\Component\Assertion\Handle;
 
+$host = '${drupal.db.host}';
+$port = '${drupal.db.port}';
+
+// If DDEV_PHP_VERSION is not set but IS_DDEV_PROJECT *is*, it means we're running (drush) on the host,
+// so use the host-side bind port on docker IP
+if (empty(getenv('DDEV_PHP_VERSION') && getenv('IS_DDEV_PROJECT') == 'true')) {
+  $host = "127.0.0.1";
+  $port = 53317;
+}
+
```

Comment:
Where did this snippet come from again?

Reply:
I removed this as it seems to allow running Drush on your host which is cool but doesn't really work with multisite Drush aliases.

Reply:
Maybe not...

Reply:
I think we can leave this out.

Reply:
a generic ddev setup creates it, but then i saw somewhere that multisite shouldn't use ddev defaults for this and instead copy out and paste it for use elsewhere. we have `disable_settings_management` turned on